woocommerce/woocommerce
# Remove 'List price' and 'Sale price' fields from the General tab

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #39644



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure feature `New product editor` is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Go to `Products` > `Add new`
3. Under the `General` tab the `Basic details` section should no longer have the `List price` and the `Sale price` fields

Before 
<img width="724" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/fe6e1403-f404-4f96-bf85-7f58e47155db">


After 
<img width="774" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/4bf1deb3-65c1-4237-a9b9-ddf2e7fb5dfe">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
